_docs_v5/1_getting_started/requirements.md

```diff
@@ -40,19 +43,16 @@ The COSMOS system uses several terms that are important to understand. The follo
 
 ## Overall Architecture and Context Diagram
 
-The following diagram shows how the applications that make up the COSMOS system relate to each other and to the targets that COSMOS is controlling.
+The following diagram shows the COSMOS 5 architecture.
 
-<img src="{{site.baseurl}}/img/architecture.png" alt="COSMOS Architecture">
+![COSMOS Architecture]({{site.baseurl}}/img/v5/architecture.png)
```

Comment:
Reused the front page architecture diagram to fix the broken link to the old COSMOS 4 diagram.